visualization/app/codeCharta/ui/legendPanel/legendPanel.component.scss

```diff
@@ -65,6 +66,16 @@ legend-panel-component {
 			.set-left-margin {
 				margin-left: 18px;
 			}
+
+			.marked-package {
+				overflow: hidden;
+				vertical-align: middle;
+				direction: rtl;
```

Comment:
The direction should probably not be changed. This is normally only necessary for languages that are read right to left instead of left to right. This should not be the case here.

https://developer.mozilla.org/en-US/docs/Web/CSS/direction

Reply:
There is no css property for text-overflow: ellipsis left. People on stackoverflow suggest the "direction-hack" as well. https://stackoverflow.com/a/9793669